jlm/mlir/frontend/MlirToJlmConverter.cpp

```diff
@@ -91,22 +94,55 @@ MlirToJlmConverter::ConvertBlock(::mlir::Block & block, rvsdg::Region & rvsdgReg
 
   // Create an RVSDG node for each MLIR operation and store each pair in a
   // hash map for easy lookup of corresponding RVSDG nodes
```

Comment:
This comment should be changed to reflect the hash map being on value-granularity, not operation granularity